### PR TITLE
Translate quiz 11 and add test

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#24 by:
1. Adding subtitles and changing id's to each exercise.
2. Translating everything from Portuguese to Spanish.
3. Formatting Python code with `black` using `--line-length 120`.
4. Adding a web test for exercise 3.

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

https://user-images.githubusercontent.com/80920858/121787226-7f80b980-cb8a-11eb-9b46-b9619efdfcb0.mp4

### CODE REVIEW

**Review by angelasofiaremolinagutierrez — APPROVED** [review]
LGTM! 👍

File: _sources/quiz/Quiz11.rst [hunk 1]
```diff
@@ -7,183 +7,225 @@ Quiz - 11
...
-   Para um inteiro n retorna uma string na forma 'Número de donuts: <n>' |br|
-   onde n é o valor passado como argumento. |br|
-   Caso n >= 10 devo retornar 'muitos' em lugar do número. |br|
-   donuts(5) returns 'Número de donuts: 5' |br|
-   donuts(23) returns 'Número de donuts: muitos' |br|
+   Desarrolle la función ``cuantas_donas`` que toma a ``n``, un número entero 
+   positivo, como parámetro, y devuelve una cadena de forma ``"Número de donas: n"``, 
+   donde ``n`` es el valor que se le pasó a la función como argumento. No obstante, 
+   si ``n`` >= 10, ``cuantas_donas`` devolverá ``"muchas"`` en vez de ``n``. |br| |br|
+   Ejemplos:
```
Comment (y-martinez) [inline _sources/quiz/Quiz11.rst:20]:
```suggestion
   Ejemplos: |br|
```

**Review by y-martinez — APPROVED** [review]
I left a little suggestion, but in general LGTM

File: _sources/quiz/Quiz11.rst [hunk 2]
```diff
@@ -221,7 +221,7 @@ Ejercicio 6
        def testOne(self):
    
            self.assertEqual(
-               contar_ocurrencias("ana y mariana gostan de bananas", "ana"),
+               contar_ocurrencias("ana y mariana gozan de bananas", "ana"),
```
Comment (NicolasSandoval) [inline _sources/quiz/Quiz11.rst:224]: I think the translation did not work well.
### Suggested change options:

- "ana y mariana compraron bananas", or
- maybe you should change the fruit to something like 'a ... les gustan las manzanas',or
-  just change it entirely.

Comment (LeoCumpli21) [reply _sources/quiz/Quiz11.rst]: Done :)

**Review by NicolasSandoval — APPROVED** [review]

**Pull request merged**